modules/ROOT/pages/configuring-mule-stacktraces.adoc

```diff
@@ -0,0 +1,84 @@
+= Configuring Mule Stacktraces
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint, studio, mule, on premises, on premise, amc, logs, stacktraces, verbose
+
+By default Mule does not show stacktraces and even filters out some internal class
+ references from stacktraces to  produce a more readable output when it does show them.
+This behavior can be controlled in one of two ways:
+
+* Command-line arguments
+
+* Setting properties in Mule Studio
+
+== Command-Line Arguments
+
+Two command-line properties let you control this behavior using `-M-DpropertyName`:
+
+* `mule.verbose.exceptions` - If set to *true*, Mule shows filtered stacktraces.
```

Comment:
i had the impression studio had this enabled by default. We should create a JIRA in STUDIO for this.

Reply:
Done